Process.c

```diff
@@ -706,6 +707,13 @@ void Process_printLeftAlignedField(RichString* str, int attr, const char* conten
    RichString_appendChr(str, attr, ' ', width + 1 - columns);
 }
 
+static inline bool is_dynamicColumn( const ProcessList* pl, int key) {
+   const DynamicColumn* column = Hashtable_get(pl->dynamicColumns, key);
+   if (!column)
+      return 1;
+   return 0;
```

Comment:
There are constants ``true` and `false` in `stdbool.h` for this.

Reply:
@BenBE Done.